apps/web/app/editor/page.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Editor from "../../components/editor/advanced-editor";
```

Comment:
You could use the alias: `import Editor from "@/components/editor/advanced-editor";`